packages/cli/src/ceramic-daemon.ts

```diff
@@ -935,6 +941,13 @@ export class CeramicDaemon {
     })
   }
 
+  async stopIndexingModelData(req: Request, res: Response): Promise<void> {
+    await this._processAdminModelsMutationRequest(req, res, {
+      type: 'modelIDs',
```

Comment:
the behavior of both endpoints is still the same, so splitting it out to a new method didn't really do anything.  This would need to be of type `modelData` to use the new object-based API structure and get the future-proofing that was the motivation for this change

Reply:
Although we only need stream id's for stop indexing, you want to pass it in as an object, in case there's additional information we may want to pass to stop indexing at a later time?

Reply:
Yes exactly, see my earlier comment: https://github.com/ceramicnetwork/js-ceramic/pull/2844#discussion_r1242701934

We can still only pass model streamids, but we can do it in the object form (e.g. `[{streamId: "<modelAStreamID>"}, {streamId: "<modelBStreamID>"}]`) as opposed to the array of strings form (e.g. `["<modelAStreamID>", "<modelBStreamID>"]`).  That way we have the structure in place that is aligned with the other endpoints and is more extensible to be able to add additional per-model arguments in the future if we need them.